R/bambu-processReads_utilityConstructReadClasses.R

```diff
@@ -163,15 +162,18 @@ createReadTable <- function(unlisted_junctions, readGrgList,
                 mcols(unlisted_junctions)$id), sep = ","),
         start = pmin(start(readRanges), intronStartCoordinatesInt),
         end = pmax(end(readRanges), intronEndCoordinatesInt),
-        strand = readStrand,
-        confidenceType = readConfidence)
+        strand = readStrand, confidenceType = readConfidence,
+        alignmentStrand = as.character(getStrandFromGrList(readGrgList))=='+')
     ## currently 80%/20% quantile of reads is used to identify start/end sites
     readTable <- readTable %>% 
         group_by(chr, strand, intronEnds, intronStarts, confidenceType) %>% 
         summarise(readCount = n(),
         start = nth(x = start, n = ceiling(readCount / 5), order_by = start),
         end = nth(x = end, n = ceiling(readCount / 1.25), order_by = end),
-        .groups = 'drop') %>% arrange(chr, start, end) %>%
+        startSD = sd(start), endSD = sd(end), 
+        readCount.posStrand = sum(alignmentStrand, na.rm = T),
+        #readCount.sameStrand = sum(sameStrand),
+        .groups = 'drop') %>% #arrange(chr, start, end) %>%
```

Comment:
This was an artifact from when read level features (e.g sd) were calculated during the txRange() call and the readGrgList was passed through. This meant the order needed to be preserved. Since this isn't done anymore this doesn't need to be removed anymore so I have uncommented it. 
Effected to memory - N/A
Effects to microbenchmark - isore.constructReadClasses() 6 seconds faster

Reply:
bbc2f8e